### PR TITLE
Update Tap Iceberg to add AWS configuration support

### PR DESCRIPTION
**Pull request by BTheunissen**

### CODE REVIEW

File: _data/meltano/extractors/tap-iceberg/shaped-ai.yml [hunk 1]
```diff
@@ -47,22 +47,30 @@ settings:
...
   label: Catalog Type
   name: catalog_type
-- description: The URI of the Iceberg catalog
-  kind: string
-  label: Catalog Uri
+- description: The URI of the catalog to connect to.
+  label: Catalog URI
   name: catalog_uri
+- description: Additional properties to pass to the catalog connection.
+  label: Catalog Properties
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-iceberg/shaped-ai.yml:60]: Should this be 

```suggestion
  kind: object
  label: Catalog Properties
```

File: _data/meltano/extractors/tap-iceberg/shaped-ai.yml [hunk 2]
```diff
@@ -47,22 +47,30 @@ settings:
...
+  name: catalog_properties
+- description: The access key of the AWS Glue Data Catalog.
+  label: Access Key
+  name: client_access_key_id
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-iceberg/shaped-ai.yml:64]:
```suggestion
  name: client_access_key_id
  sensitive: true
```

File: _data/meltano/extractors/tap-iceberg/shaped-ai.yml [hunk 3]
```diff
@@ -47,22 +47,30 @@ settings:
...
+- description: The secret access key of the AWS Glue Data Catalog.
+  label: Secret Access Key
+  name: client_secret_access_key
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-iceberg/shaped-ai.yml:67]:
```suggestion
  name: client_secret_access_key
  sensitive: true
```

File: _data/meltano/extractors/tap-iceberg/shaped-ai.yml [hunk 4]
```diff
@@ -47,22 +47,30 @@ settings:
...
+- description: The session token of the AWS Glue Data Catalog.
+  label: Session Token
+  name: client_session_token
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-iceberg/shaped-ai.yml:70]:
```suggestion
  name: client_session_token
  sensitive: true
```

**Comment by BTheunissen** [comment]
@edgarrmondragon Thanks, fixed up if you can take another look

**Review by edgarrmondragon — APPROVED** [review]

**Pull request merged**